torch/onnx/symbolic_opset14.py

```diff
@@ -0,0 +1,90 @@
+# EDITING THIS FILE? READ THIS FIRST!
+# see Note [Edit Symbolic Files] in symbolic_helper.py
+
+# This file exports ONNX ops for opset 14
+import torch
+from torch.onnx.symbolic_opset9 import _var_mean
+
+import torch.onnx.symbolic_helper as sym_help
+from torch.onnx.symbolic_helper import parse_args
+from torch.onnx.symbolic_helper import _block_list_in_opset
+
+# Note [ONNX operators that are added/updated in opset 14]
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# New operators:
+#   HardSwish, Trilu
+#
+# Updated operators:
+#   Reshape
+#   Add, Sub, Mul, Div
+#   GRU, LSTM, RNN
+#   BatchNorm, Cumsum, Relu
+
+block_listed_operators = [
+]
+
+for block_listed_op in block_listed_operators:
+    vars()[block_listed_op] = _block_list_in_opset(block_listed_op)
+
+@parse_args("v")
+def hardswish(g, self):
+    return g.op("HardSwish", self)
+
+@parse_args("v", "i")
+def tril(g, self, diagonal, out=None):
+    k = g.op("Constant", value_t=torch.tensor(diagonal, dtype=torch.int64))
+    return g.op("Trilu", self, k, upper_i=0)
+
+@parse_args("v", "i")
+def triu(g, self, diagonal, out=None):
+    k = g.op("Constant", value_t=torch.tensor(diagonal, dtype=torch.int64))
+    return g.op("Trilu", self, k, upper_i=1)
+
+@parse_args("v", "v")
+def reshape(g, self, shape):
+    shape = sym_help._maybe_get_const(shape, "is")
+    if not sym_help._is_value(shape):
+        shape = g.op("Constant", value_t=torch.LongTensor(shape))
+    return g.op("Reshape", self, shape, allowzero_i=0)
```

Comment:
i think for converting reshape from pytorch, the allowzero should be set to 1. Because pytorch reshape does not have semantic of "0 in shape preserving original value", 0 literally means 0. However, the blocker is that other symbolic functions may call `reshape` with 0 in shape and expect ONNX semantic. Please check if that's the case, and think of a proper design.

And in addition, may need to revise all the call sites in symbolic functions in other opsets, to invoke `reshape` based on actual opsets.